### PR TITLE
Add doc on gradle project properties and cmd line args

### PR DESCRIPTION
**Pull request by shahzaibj**

### CODE REVIEW

**Review by AdamBJohnsonx — APPROVED** [review]
Could we put a link to this file from the root README.md?

**Review by tanmaymanolkar1 — APPROVED** [review]

**Comment by shahzaibj** [comment]
> Could we put a link to this file from the root README.md?

Addressed here: https://github.com/AzureAD/android-complete/pull/68/commits/c4040d4dd69e5a41922fb90f247d4873a844a004

File: docs/ProjectBuild/gradle_project_properties.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+![Android Studio Command Line Parameters](command_line_args.png "Android Studio Command Line Parameters")
+
+## Which properties / arguments are supported across which of our android-complete modules?
+
+We currently support the following gradle properties via command line across the following modules:
+
+| Property               | Declaring Module                              | Description                                                  |
+| ---------------------- | --------------------------------------------- | ------------------------------------------------------------ |
+| labSecret              | testutils                                     | Enables access to [Lab Api](../Automation/labsetup.md) to get test accounts |
```
Comment (iamgusain) [inline docs/ProjectBuild/gradle_project_properties.md:31]: it will be good to add another column that shows example/supported values for these properties

Comment (shahzaibj) [reply docs/ProjectBuild/gradle_project_properties.md:31]: I've done it for brokerSource here: https://github.com/AzureAD/android-complete/pull/68/commits/dd4827c0f6fab443353f9e3111e6d9255cae1c34

For the rest of these, the supported values are self explanatory such as msalVersion has to be a valid msal version...lab secret needs to a client secret value etc...

I do have an example of how to pass these at the bottom of this doc....for all these the mechanism to supply them is the same so I think one example should be sufficient.

**Pull request merged**